### PR TITLE
Cu 2kb6yf7 fix integration issues

### PR DESCRIPTION
**Pull request by morsdyce**

### CODE REVIEW

File: src/command.ts [hunk 1]
```diff
@@ -14,5 +14,7 @@ export const onCommandReceived = async (command: Command, responseTopic: string)
 
   const commandStatus = await executeCommand(command);
 
+  console.log({ commandStatus, responseTopic, correlationData: command.id });
```
Comment (Chikel) [inline src/command.ts:17]: Do we wanna add a prefix, to understand the context of the log? (e.g. onCommandRecieved - rest...)

File: src/firmware-version.ts [hunk 2]
```diff
@@ -3,14 +3,15 @@ import { mqttClient } from './helpers/mqtt.js';
...
-export const isFirmwareVersionMessage = (topic: string) => /^v1\/device\/.*\/space_info$/.test(topic);
+export const isFirmwareVersionMessage = (topic: string) => /^v1\/device\/.*\/firmware$/.test(topic);
 
 export const onFirmwareVersionMessage = async (firmwareVersion: FirmwareVersion) => {
   console.log('example firmware version compare, firmware retrieving ');
 
   const serverFirmwareVersion = firmwareVersion.latest_fw_version;
   const deviceFirmwareVersion = getDeviceFirmwareVersion();
 
+  console.log({ serverFirmwareVersion, deviceFirmwareVersion });
```
Comment (Chikel) [inline src/firmware-version.ts:14]: Same here.

File: src/mqtt-subscriber.ts [hunk 3]
```diff
@@ -30,7 +30,7 @@ export const listenToMessages = () => {
         await onFirmwareVersionMessage(message as FirmwareVersion);
       }
     } catch (error) {
-      console.log(`Failed parsing message for topic: ${topic}`);
+      console.log(`Failed parsing message for topic: ${topic}`, payload.toString());
```
Comment (Chikel) [inline src/mqtt-subscriber.ts]: console.error?

File: src/command.ts (hunk 1 above)
Comment (morsdyce) [reply src/command.ts:17]: You already have context from the event received in the subscriber

**Review by ilyagelman — APPROVED** [review]

**Pull request merged**